### PR TITLE
OTG TE - 9.1 Changing the test not to update the flows dynamically

### PR DESCRIPTION
**Pull request by ANISH-GOTTAPU**
Changing the test not to update the flows dynamically

### CODE REVIEW

File: feature/experimental/gribi/otg_tests/fib_failed_due_to_hw_res_exhaust_test/fib_failed_due_to_hw_res_exhaust_test.go [hunk 1]
```diff
@@ -64,6 +62,12 @@ const (
 	tolerance                 = 50
 	plenIPv4                  = 30
 	plenIPv6                  = 126
+	fibPassedDstRoute         = "203.0.112.0"
+	fibFailedDstRoute         = "203.0.112.1"
```
Comment (bkreddy143) [inline feature/experimental/gribi/otg_tests/fib_failed_due_to_hw_res_exhaust_test/fib_failed_due_to_hw_res_exhaust_test.go:66]: it is not guaranteed that very next route after BGP routes injection will fail. we need to keep injecting gRIBI routes and track the first failed and same needs to be used to create the failed steam

File: feature/experimental/gribi/otg_tests/fib_failed_due_to_hw_res_exhaust_test/fib_failed_due_to_hw_res_exhaust_test.go [hunk 2]
```diff
@@ -297,50 +326,15 @@ func TestFibFailDueToHwResExhaust(t *testing.T) {
...
 	args.otg.StopTraffic(t)
 
-	verifyTraffic(t, args, flow1ipv4.Name(), !wantLoss)
-	/*
-		if !deviations.GRIBISkipFIBFailedTrafficForwardingCheck(args.dut) {
-			verifyTraffic(t, args, flow2ipv4.Name(), wantLoss)
-		}
-	*/
+	verifyTraffic(t, args, fibPassedTraffic, !wantLoss)
+	verifyTraffic(t, args, fibFailedTraffic, wantLoss)
```
Comment (bkreddy143) [inline feature/experimental/gribi/otg_tests/fib_failed_due_to_hw_res_exhaust_test/fib_failed_due_to_hw_res_exhaust_test.go:337]: Leave this deviation as it as unless there is confirmation from vendor who added this. 
'''
if !deviations.GRIBISkipFIBFailedTrafficForwardingCheck(args.dut) {
			verifyTraffic(t, args, flow2ipv4.Name(), wantLoss)
		}
'''

**Review by bkreddy143 — APPROVED** [review]
Looks Good .

**Comment by ANISH-GOTTAPU** [comment]
@jasdeep-hundal : Can you please review it and let me know if any changes needed

**Review by nsadhasivam — APPROVED** [review]
LGTM!

**Pull request merged**